frontend/src/libs/components/app/app.tsx

```diff
@@ -16,6 +33,17 @@ const App: React.FC = () => {
     }
   }, [isRoot, dispatch]);
 
+  useEffect(() => {
+    if (!localStorage.getItem(StorageKey.TOKEN)) {
```

Comment:
Let's add an wrapper service for using localStorage

Reply:
we already have it - storage package